src/main/resources/db/migration/v0.37.0.60__expression_atlas-load.sql

```diff
@@ -0,0 +1,82 @@
+insert into bulkloadgroup (id, name, internal, obsolete, dbdatecreated)
```

Comment:
Will need to rename this file as another went in ahead of it:

0.37.0.61

Reply:
Yes.